Platform/CommonBoardPkg/AcpiTables/Dsdt/FwuWmi.asl

```diff
@@ -0,0 +1,115 @@
+/**@file
+
+  Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+Device(DWMI)
+{
+  // PNP0C14 is PNP ID assigned to WMI mapper
+  Name(_HID, EISAID("PNP0C14"))
+  Name(_UID, 0x0)
+
+  Name(_WDG, Buffer(){
+    // {44FADEB1-B204-40F2-8581-394BBDC1B651}
+    // This user defined GUID for providing interfaces for WQ00/WS00
+    0xb1, 0xde, 0xfa, 0x44, 0x04, 0xb2, 0xf2, 0x40, 0x85, 0x81, 0x39, 0x4b, 0xbd, 0xc1, 0xb6, 0x51,
+    0x30, 0x30,   // Object Id (00), need to match WQ00/WS00 below.
+    0x01,         // Instance Count
+    0x01,         // Flags (WMIACPI_REGFLAG_EXPENSIVE)
+
+    // {05901221-D566-11D1-B2F0-00A0C9062910}
+    // This GUID for returning the MOF data, do not change
+    0x21, 0x12, 0x90, 0x05, 0x66, 0xd5, 0xd1, 0x11, 0xb2, 0xf0, 0x00, 0xa0, 0xc9, 0x06, 0x29, 0x10, //
+    0x5A, 0x5A,   // Object ID (ZZ), need to match WQZZ below.
+    0x01,         // Instance Count
+    0x00          // Flags
+  })
+
+  Method(WQ00, 1)
+  {
+    Return (FWUC(0, 0))
+  }
+
+  Method(WS00, 2)
```

Comment:
Arg0 is not used. Still necessary to take 2 args?

Reply:
It is defined by WMI calling convention.  Arg0 is instance.